.github/workflows/test.yaml

```diff
@@ -14,16 +14,14 @@ jobs:
 
       - name: Run tests
         run: yarn jest --maxWorkers=2
-  # test-server:
-  #   runs-on: ubuntu-latest
-  #   env:
-  #     GOPRIVATE: "github.com/portainer"
-  #   steps:
-  #     - uses: actions/checkout@v3
-  #     - uses: actions/setup-go@v3
-  #       with:
-  #         go-version: '1.18'
-  #     - name: Run tests
-  #       run: |
-  #         cd api
-  #         go test ./...
+  test-server:
+    runs-on: ubuntu-latest
+    env:
+      GOPRIVATE: "github.com/portainer"
```

Comment:
This line is only needed for EE.  And is wrong anyway.  It isn't used so it won't break anything.